content/100-getting-started/01-quickstart-prisma-postgres.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Quickstart'
+metaTitle: 'Quickstart with TypeScript & Prisma Postgres'
+metaDescription: 'Get started with Prisma ORM in 5 minutes. You will learn how to send queries to a SQLite database in a plain TypeScript script using Prisma Client.'
+search: true
+dbSwitcher: ['sqlite', 'prisma-postgres']
+slugSwitch: /getting-started/quickstart-`
+sidebar_custom_props: { badge: '5 min' }
+---
+
+In this Quickstart guide, you'll learn how to get started from scratch with Prisma ORM and a **Prisma Postgres** database in a plain **TypeScript** project. It covers the following workflows:
+
+- Creating a [Prisma Postgres](https://prisma.io/data-platform/postgres) database
+- Schema migrations and queries (via [Prisma ORM](https://www.prisma.io/orm))
+- Connection pooling and caching (via [Prisma Accelerate](https://prisma.io/data-platform/accelerate))
+- Real-time database change events (via [Prisma Pulse](https://prisma.io/data-platform/pulse))
+
+## Prerequisites
+
+To successfully complete this tutorial, you need:
+- a [Prisma Data Platform](https://console.prisma.io/) (PDP) account
+- Node.js v16.13.0 or higher (learn more about [system requirements](/orm/reference/system-requirements))
+
+## 1. Set up a Prisma Postgres database in the PDP Console
+
+Follow these steps to create your Prisma Postgres database:
+
+1. Log in to [Prisma Data Platform](https://console.prisma.io/).
+1. In a [workspace](/platform/about#workspace) of your choice, click the **New project** button.
+1. Type a name for your project in the **Name** field, e.g. **hello-ppg**.
+1. In the **Prisma Postgres** section, click the **Get started** button.
+1. In the **Region** dropdown, select the region that's closest to your current location, e.g. **US East (N. Virginia)**.
+1. Click the **Create project** button.
+
+At this point, you'll be redirected to the **Dashboard** where you will need to wait for a few seconds while the status of your database changes from **`PROVISIONING`**, to **`ACTIVATING`** to **`CONNECTED`**.
+
+Once the green **`CONNECTED`** label appears, your database is ready to use!
+
+## 2. Download example and install dependencies
+
+Copy the `try-prisma` command that's shown in the Console, paste it into your terminal and execute it.
+
+For reference, this is what the command looks like (note that the `__YOUR_DATABASE_CONNECTION_STRING__` placeholder must be replaced with _your_ actual database connection string):
+
+```
+npx try-prisma@latest
+  --template prisma-starter
+  --connection-string __YOUR_DATABASE_CONNECTION_STRING__
+  --name hello-prisma-postgres
+  --install npm
+  --path .
```

Comment:
```suggestion
npx try-prisma@latest \
  --template prisma-starter \
  --connection-string __YOUR_DATABASE_CONNECTION_STRING__ \
  --name hello-prisma-postgres \
  --install npm \
  --path .
```

@nikolasburk, I tried this, and we need some newline escapes. I also got an error for now for --connection-string:

```
❯ npx try-prisma@latest \
  --template prisma-starter \
  --connection-string __YOUR_DATABASE_CONNECTION_STRING__ \
  --name hello-prisma-postgres \
  --install npm \
  --path .
Cannot run command, you made some mistakes:

Unknown flag "connectionString"
X Unknown flag "path"
```

Reply:
Should be `database-url` and `path` is now removed.
```suggestion
npx try-prisma@latest \
  --template prisma-starter \
  --database-url __YOUR_DATABASE_CONNECTION_STRING__ \
  --name hello-prisma-postgres \
  --install npm
```